### DISCUSSION TITLE
Newbie question: handling Effects

### DISCUSSION THREAD

**Discussion by Pharmudtix** [Mezryss/FVTT-Genesys#173] [Q&A]
So I've spent the past few days entering the text descriptions from Twilight Imperium for most of the item types. I'm statting stuff now and attaching qualities, but I'm clueless as to how to do the effects. I know where to enter them, but after that...I'm at a complete loss.

Is there a community "cheat sheet/how to" somewhere that breaks it all down?  For example, how do I add ignore Soak on a weapon with Pierce?

Any help would be much appreciated.

Feel free to explain to me like I'm old and dumb, because I am.

**Comment by Assembling-Kings** [comment]
So currently the type of effects that can be done is very limited; something like ignoring Soak is not possible. The most you can do is modify some of the stats (or the dice pool) of the character with the effect. In the future I'll add more support for stuff like that but that's not something that will happen soon. With all that said, here's some general pointers.
- Like I mentioned, for a simple stat modification just select the stat and "Add" a positive or negative value. That should work fine unless it's for a field that can be easily edited (like say, current wounds) since those will behave abnormally when edited.
- Modifying the dice pool can only be done for specific skills. Follow the proper documentation to do that.
- An item/archetype/etc that has an effect will make a copy of its effects and add them to the actor it's dropped upon. This means that modifying the effects on the item after dropping it won't be carried over to the character. Instead edit the copy that is on the character to signify any changes.
- Stat effects are applied multiple times if added to a ranked talent, depending on the number of ranks on said talent. That is not the case if it's a dice pool effect; it will only be applied once.

And I think that's the gist of it. Any further questions please let me know.

**Comment by DNightone** [comment]
I'll put this here since I think it's more a follow up question:

How does the "duration" for effects work - How would I add a turn number to an temporary effect like disorient 1 or a crtical injury effect that last only for one turn?

I tried the "Durration tab" but couldn't figure the correct set up...   (may be I am also old and dumb)

**Reply by Assembling-Kings** [reply]
Actually, I myself don't even know how it works. I haven't looked at that part of Foundry's code so I'm unsure what the default implementation does. This is not something I plan to look at too deeply until I better implement consumable items.
